docs/upgrade/3.5.0.rst

```diff
@@ -0,0 +1,115 @@
+so that they no longer refer to specifically to pages. Ultimately, publishing actions could apply
+to any kind of content, and this is a step in that direction.
+
+
+*******************
+What's new in 3.5.0
+*******************
+
+Improvements and new features
+=============================
+
+* structure board now decoupled from page rendering
+* ``Page`` model decoupled from the site navigation
+* Page copy between sites
+* better behaviour of the language chooser for published/unpublished languages
+* improved handling, refactored code for language fallbacks
+* improved repr for ``Page``, ``Title``, ``Placeholder`` and ``CMSPlugin`` models
+* generic publishing controls no longer refer to "page"
+* improved documentation
+
+
+Bug Fixes
+=========
+
+This release fixes:
+
+* a bug in which Placeholder ordering could be incorrect
+* a Page template settings permission issue (failed to check for "Change advanced settings
+  permission")
+* a bug allowing Pages to be pasted without the correct translations for the target site
+* a bug that prevented users from seeing the welcome screen when debug is off
+* a bug allowing aliased plugins to render even if their host page was unpublished
+* a bug where focusing inputs in modal would require two clicks in some browsers
+* minor issues with initialisation of interface widgets.
+* minor clipboard bugs
+* numerous minor issues with rendering/interaction of the structureboard
+
+
+Removal of deprecated functionality
+===================================
+
+Previously deprecated functionality has been removed:
+
+* Menu modules can no longer be named ``menus.py`` (use ``cms_menus.py``).
+* The ``cms.utils.django_load.py`` module has been removed (in favour of standard Django helpers)
+* Support for Django Reversion has been removed.
+* The ``urls`` and ``menus`` attributes are no longer supported on ``CMSApp`` (apphook) classes.
+* ``Page.revision_id`` has been removed
+* Deprecated content creation wizard settings have been removed.
+
+
+Backward-incompatible changes
+=============================
+
+* The home page is no longer automatically the root page in the tree (since there is no longer
+a page tree). Instead, the home page is managed via a page setting.
+* Pages no longer the source of the navigation tree. Previously, ordered pages could be obtained
+  via ``Page.object.order_by('path')``; the equivalent is now
+  ``Page.object.order_by('node__path')``.
+* Pages are no longer ordered by path. For pages according to tree ordering, use
+  ``order_by==node__path`` in filters.
+* Pages no longer have a ``site`` field page. Whereas previously you could use ``filter(site=id)``,
+  now use ``filter(node__site==id)``.
+* Pages no longer have a ``parent`` field. Instead a ``parent`` property now returns the new
+  ``parent_page`` attribute, which relies on the node tree.
+* Pages can no longer have a 'pending' publishing state. A data migration,
```

Comment:
Not fully accurate.
Pages can still be pending if their parent is unpublished.
They can no longer be pending if they've never been published before.